src/main/java/gregtech/common/covers/CoverFluidVoiding.java

```diff
@@ -79,16 +78,15 @@ public ModularPanel buildUI(SidedPosGuiData guiData, PanelSyncManager guiSyncMan
 
     @Override
     protected ParentWidget<?> createUI(GuiData data, PanelSyncManager syncManager) {
-        var isWorking = new BooleanSyncValue(this::isWorkingEnabled, this::setWorkingEnabled);
+        // noinspection DuplicatedCode
+        BooleanSyncValue isWorking = new BooleanSyncValue(this::isWorkingEnabled, this::setWorkingEnabled);
 
         return super.createUI(data, syncManager)
                 .child(Flow.row().height(18).widthRel(1f)
                         .marginBottom(2)
                         .child(new ToggleButton()
                                 .value(isWorking)
-                                .overlay(IKey.dynamic(() -> IKey.lang(this.isWorkingAllowed ?
-                                        "behaviour.soft_hammer.enabled" :
-                                        "behaviour.soft_hammer.disabled").get())
+                                .overlay(createEnabledKey(() -> this.isWorkingAllowed)
```

Comment:
this changes the lang key used, should `behaviour.soft_hammer.enabled` be removed if it's not used?